### PR TITLE
ftu-header

### PR DESCRIPTION
**Pull request by rchavan10**

### CODE REVIEW

**Comment by rchavan10** [comment]
I ran lint on all files still the format check test is failing.

File: libs/components/molecules/src/lib/header/header.component.html [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<a class="logo-link" [routerLink]="['/']"><img class="logo-image" [src]="logo" alt="" /></a>
+<p class="header-text-1">Human Reference Atlas</p>
```
Comment (axdanbol) [inline libs/components/molecules/src/lib/header/header.component.html:2]: Should be part of the link
header-text-1 -> product-title
Use h1, h2, or whatever fits

File: libs/components/molecules/src/lib/header/header.component.html [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+<p class="header-text-2">Functional Tissue Unit Explorer</p>
```
Comment (axdanbol) [inline libs/components/molecules/src/lib/header/header.component.html:3]: Use h1, h2, or whatever fits
header-text-2 -> app-title

File: libs/components/molecules/src/lib/header/header.component.html [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+<div class="filler"></div>
+<img class="info-image" [src]="info" alt="" />
```
Comment (axdanbol) [inline libs/components/molecules/src/lib/header/header.component.html:5]: Put icon url into html as it is not going to change
Move into link

File: libs/components/molecules/src/lib/header/header.component.html [hunk 4]
```diff
@@ -0,0 +1,7 @@
...
+<p class="header-text-3"> <a [routerLink]="['/']">About</a></p>
```
Comment (axdanbol) [inline libs/components/molecules/src/lib/header/header.component.html:6]: Use span
header-text-3 -> about

File: libs/components/molecules/src/lib/header/header.component.scss [hunk 5]
```diff
@@ -0,0 +1,50 @@
+:host {
+  display: flex;
+  // flex-direction: row;
```
Comment (axdanbol) [inline libs/components/molecules/src/lib/header/header.component.scss:3]: Remove commented out code

File: libs/components/molecules/src/lib/header/header.component.scss [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+  // align-items: center;
+  // padding: 1rem 2rem;
+  background: #ffffff;
+  border-bottom: 1px solid #e2e7e9;
+  font-family: 'Inter';
```
Comment (axdanbol) [inline libs/components/molecules/src/lib/header/header.component.scss:8]: Remove font-family, etc.

File: libs/components/molecules/src/lib/header/header.component.scss [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+  font-style: normal;
+  justify-content: space-between;
```
Comment (axdanbol) [inline libs/components/molecules/src/lib/header/header.component.scss:10]: Remove

File: libs/components/molecules/src/lib/header/header.component.spec.ts [hunk 8]
```diff
@@ -0,0 +1,21 @@
...
+  beforeEach(() => {
+    shallow = new Shallow(HeaderComponent);
+  });
+
+  it('renders images correctly', async () => {
+    const { instance } = await shallow.render({
+      bind: { logo: logoSrc, info: infoSrc },
+    });
+
+    expect(instance.logo).toBe(logoSrc);
```
Comment (axdanbol) [inline libs/components/molecules/src/lib/header/header.component.spec.ts:18]: `await expect(shallow.render()).resolves.toBeDefined()`

File: libs/components/molecules/src/lib/header/header.component.ts [hunk 9]
```diff
@@ -0,0 +1,15 @@
...
+  selector: 'hra-header',
+  standalone: true,
+  imports: [CommonModule],
+  templateUrl: './header.component.html',
+  styleUrls: ['./header.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class HeaderComponent {
+  @Input() logo = '';
+  @Input() info = '';
```
Comment (axdanbol) [inline libs/components/molecules/src/lib/header/header.component.ts:14]: Remove

File: libs/components/molecules/src/lib/header/header.component.ts [hunk 10]
```diff
@@ -0,0 +1,15 @@
...
+@Component({
+  selector: 'hra-header',
+  standalone: true,
+  imports: [CommonModule],
+  templateUrl: './header.component.html',
+  styleUrls: ['./header.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class HeaderComponent {
+  @Input() logo = '';
```
Comment (axdanbol) [inline libs/components/molecules/src/lib/header/header.component.ts:13]: Add productTitle and appTitle inputs

**Review by axdanbol — CHANGES_REQUESTED** [review]

File: libs/components/molecules/src/lib/header/header.component.scss (hunk 6 above)
Comment (rchavan10) [reply libs/components/molecules/src/lib/header/header.component.scss:8]: removing the font-family causes the text to revert back to default settings and not 'Inter'

File: libs/components/molecules/src/lib/header/header.component.scss (hunk 7 above)
Comment (rchavan10) [reply libs/components/molecules/src/lib/header/header.component.scss:10]: removed font-style and justify-content.

File: libs/components/molecules/src/lib/header/header.component.html [hunk 11]
```diff
@@ -0,0 +1,11 @@
+<a class="product-logo-link" [routerLink]="['/']">
+  <img class="product-logo-image" [src]="productLogoUrl" alt="" />
+  <h2 class="product-title">{{ productTitle }}</h2>
+</a>
+
+<h2 class="app-title">{{ appTitle }}</h2>
+<div class="filler"></div>
+<a [routerLink]="['/']">
+  <img class="info-image" [src]="'/assets/icons/info.svg'" alt="" />
```
Comment (axdanbol) [inline libs/components/molecules/src/lib/header/header.component.html:9]: Use mat-icon instead `<mat-icon>info</mat-icon>`

File: libs/components/molecules/src/lib/header/header.component.scss (hunk 6 above)
Comment (axdanbol) [reply libs/components/molecules/src/lib/header/header.component.scss:5]: That will automatically be fixed once I've merged my theming changes today, so you can remove this

File: libs/components/molecules/src/lib/header/header.component.stories.ts [hunk 12]
```diff
@@ -0,0 +1,23 @@
...
+const Template: Story<HeaderComponent> = (args: HeaderComponent) => ({
+  props: args,
+});
+
+export const Primary = Template.bind({});
+Primary.args = {
+  productLogoUrl: '/assets/icons/logo-icon.svg',
+  productTitle: 'Human Reference Atlas',
+  appTitle: 'Functional Tissue Unit Explorer',
+};
```
Comment (axdanbol) [inline libs/components/molecules/src/lib/header/header.component.stories.ts:23]: Add small and large stories with different width set. Please see me and I'll show you how

**Review by axdanbol — CHANGES_REQUESTED** [review]

**Comment by axdanbol** [comment]
Run npx nx format:write to fix formatting

**Review by axdanbol — APPROVED** [review]

**Pull request merged**